src/Browsershot.php

```diff
@@ -407,8 +430,8 @@ protected function createCommand(string $url, string $action, array $options = [
             $command['options']['ignoreHttpsErrors'] = $this->ignoreHttpsErrors;
         }
 
-        if ($this->noSandbox) {
-            $command['options']['args'] = ['--no-sandbox'];
+        if ($args = $this->getOptionArgs()) {
```

Comment:
Is this `if` necessary? `getOptionArray now always returns an array.

Reply:
No it can be safely omitted - it is equivalent to what happens in line 12 of bin/browser.js
```javascript
  args: request.options.args || []
```

Reply:
Remove it :-)